charts/yawol-controller/templates/yawol-cloud-controller.yaml

```diff
@@ -77,8 +77,11 @@ spec:
         resources:
 {{ toYaml .Values.resources.yawolCloudController | indent 10 }}
         {{- end }}
-{{- if .Values.yawolCloudController.additionalVolumeMounts }}
         volumeMounts:
+        - mountPath: /var/run/secrets/gardener.cloud/shoot/generic-kubeconfig
```

Comment:
I would like to remove the settings related to gardener from here.
Because yawol also works without gardener.